services/workflows-service/src/alert/alert.controller.external.ts

```diff
@@ -135,7 +137,17 @@ export class AlertControllerExternal {
   async decision(
     @common.Body() { alertIds, decision }: AlertDecisionDto,
     @CurrentProject() currentProjectId: TProjectId,
+    @UserData() authenticatedAssignee: AuthenticatedEntity,
   ): Promise<TBulkAssignAlertsResponse> {
+    // Assign alerts to the authenticated assignee
+    if (authenticatedAssignee?.user?.id) {
+      await this.service.updateAlertsAssignee(
+        alertIds,
+        currentProjectId,
+        authenticatedAssignee.user.id,
+      );
+    }
```

Comment:
@alonp99 @liorzam 👀

Reply:
its mean you will always assign alerts to you... the one who execute the request?